docker-compose.yml

```
@@ -0,0 +1,14 @@
+services:
+  postgres-db:
+    container_name: postgres-db
+    image: postgres:15.10
+    platform: linux/x86_64
+    restart: always
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER:-root}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-rootpassword}
+      - POSTGRES_DB=${POSTGRES_NAME-habits:-todo}
+    ports:
+      - "${POSTGRES_PORT:-5432}:5432"
+    volumes:
+      - ${POSTGRES_DATA_DIR:-./.docker/data/postgres}:/var/lib/postgresql/data
```

o diretório .docker ja faz parte do contexto atual, não precisa do `./` no começo
```suggestion
      - ${POSTGRES_DATA_DIR-.docker/data/postgres}:/var/lib/postgresql/data
```